### PR TITLE
Look for perf-java-flames on path in MultiJVM perf support

### PR DESCRIPTION
**Pull request by chbatey**
Was playing with the perf support and found this more convenient

### CODE REVIEW

**Review by ktoso — APPROVED** [review]
LGTM from my side

File: akka-multi-node-testkit/src/main/scala/akka/remote/testkit/PerfFlamesSupport.scala [hunk 1]
```diff
@@ -45,13 +50,10 @@ private[akka] trait PerfFlamesSupport { _: MultiNodeSpec ⇒
...
 
-  def perfJavaFlamesPath: String =
-    "/home/ubuntu/perf-java-flames"
-
   def isPerfJavaFlamesAvailable: Boolean = {
-    val isIt = new File(perfJavaFlamesPath).exists()
-    if (!isIt) println(s"WARN: perf-java-flames not available under [$perfJavaFlamesPath]! Skipping perf profiling.")
+    import scala.sys.process._
+    val isIt = (s"type $Command" !) == 0
+    if (!isIt) println(s"WARN: perf-java-flames on the path! Skipping perf profiling.")
```
Comment (raboof) [inline akka-multi-node-testkit/src/main/scala/akka/remote/testkit/PerfFlamesSupport.scala]: I guess this is missing a 'not'? :)

**Review by raboof — APPROVED** [review]
I didn't know we support that, cool - one log message tweak and then LGTM

File: akka-multi-node-testkit/src/main/scala/akka/remote/testkit/PerfFlamesSupport.scala (hunk 1 above)
Comment (chbatey) [reply akka-multi-node-testkit/src/main/scala/akka/remote/testkit/PerfFlamesSupport.scala]: Doh!

**Pull request merged**